TestDockerfileServer

```diff
@@ -0,0 +1,13 @@
+FROM golang:alpine
```

Comment:
After you restore the main dockerfile for the server, this can just inherit from it and make changes required for integration testing on top of that. But ideally we'd unify them and you can run the same container.